src/Console/Command/SendExport.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * Ushahidi Export Send Console Command
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Console
+ * @copyright  2018 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Console\Command;
+
+use Illuminate\Console\Command;
+
+use Ushahidi\Core\Entity\ExportJobRepository;
+
+class Webhook extends Command
```

Comment:
Should this be called something more like SendExport?

Reply:
Yep